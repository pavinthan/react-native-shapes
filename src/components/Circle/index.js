import Circle from './Circle';
import defaultProps from './defaultProps';
import propTypes from './propTypes';

Circle.propTypes = propTypes;
Circle.defaultProps = defaultProps;

export default Circle;
