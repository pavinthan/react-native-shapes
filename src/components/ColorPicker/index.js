import ColorPicker from './ColorPicker';
import defaultProps from './defaultProps';
import propTypes from './propTypes';

ColorPicker.propTypes = propTypes;
ColorPicker.defaultProps = defaultProps;

export default ColorPicker;
