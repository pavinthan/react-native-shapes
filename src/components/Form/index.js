import Form from './Form';
import defaultProps from './defaultProps';
import propTypes from './propTypes';

Form.propTypes = propTypes;
Form.defaultProps = defaultProps;

export default Form;
