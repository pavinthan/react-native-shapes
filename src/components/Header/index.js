import Header from './Header';
import defaultProps from './defaultProps';
import propTypes from './propTypes';

Header.propTypes = propTypes;
Header.defaultProps = defaultProps;

export default Header;
