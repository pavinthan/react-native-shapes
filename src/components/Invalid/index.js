import Invalid from './Invalid';
import defaultProps from './defaultProps';
import propTypes from './propTypes';

Invalid.propTypes = propTypes;
Invalid.defaultProps = defaultProps;

export default Invalid;
