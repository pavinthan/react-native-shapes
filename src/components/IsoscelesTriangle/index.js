import IsoscelesTriangle from './IsoscelesTriangle';
import defaultProps from './defaultProps';
import propTypes from './propTypes';

IsoscelesTriangle.propTypes = propTypes;
IsoscelesTriangle.defaultProps = defaultProps;

export default IsoscelesTriangle;
