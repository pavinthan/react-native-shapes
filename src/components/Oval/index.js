import Oval from './Oval';
import defaultProps from './defaultProps';
import propTypes from './propTypes';

Oval.propTypes = propTypes;
Oval.defaultProps = defaultProps;

export default Oval;
