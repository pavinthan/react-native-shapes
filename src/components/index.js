export { default as Circle } from './Circle';
export { default as ColorPicker } from './ColorPicker';
export { default as Form } from './Form';
export { default as Header } from './Header';
export { default as Invalid } from './Invalid';
export { default as IsoscelesTriangle } from './IsoscelesTriangle';
export { default as Oval } from './Oval';
export { default as Parallelogram } from './Parallelogram';
export { default as Polygon } from './Polygon';
export { default as Rectangle } from './Rectangle';
export { default as Whiteboard } from './Whiteboard';
