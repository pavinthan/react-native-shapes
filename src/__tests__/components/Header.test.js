import React from 'react';
import { Header } from '../../components';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const rendered = renderer.create(<Header />).toJSON();
  expect(rendered).toBeTruthy();
});

it('renders header component', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
