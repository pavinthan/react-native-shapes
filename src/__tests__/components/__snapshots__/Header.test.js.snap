// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header component 1`] = `
<View
  onLayout={[Function]}
>
  <View
    style={
      Array [
        Object {
          "backgroundColor": "#F8F8F8",
          "borderBottomColor": "#a7a6ab",
          "borderBottomWidth": 0.5,
          "elevation": 3,
          "flexDirection": "row",
          "height": 64,
          "justifyContent": "center",
          "left": 0,
          "paddingLeft": 6,
          "paddingRight": 10,
          "paddingTop": 18,
          "right": 0,
          "shadowColor": undefined,
          "shadowOffset": undefined,
          "shadowOpacity": undefined,
          "shadowRadius": undefined,
          "top": 0,
        },
        Object {
          "backgroundColor": "#000",
        },
      ]
    }
    virtual={undefined}
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "alignSelf": "center",
          "flex": 1,
        }
      }
      virtual={undefined}
    >
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        numberOfLines={1}
        style={
          Array [
            Object {
              "color": "#000",
              "fontFamily": "System",
              "fontSize": 17,
              "fontWeight": "700",
              "marginLeft": undefined,
              "paddingLeft": 4,
              "paddingTop": 1,
              "textAlign": "center",
            },
            Object {
              "color": "#f4f4f4",
            },
          ]
        }
        virtual={undefined}
      />
    </View>
  </View>
</View>
`;
